_2020/version-control.md

```diff
@@ -465,7 +465,13 @@ command is used for merging.
 - `git pull`: same as `git fetch; git merge`
 - `git clone`: download repository from remote
 
-# Advanced Git
+## Undo Git
+
+- `git commit --amend`: add the current commit to previous commit (only for commits not pushed)
+- `git reset HEAD <file>`: unstage a file
+- `git checkout -- <file>`: discard changes
+
+## Advanced Git
```

Comment:
I think this was supposed to be a h1. The idea was that these are advanced Git _concepts_ (and they happen to be listed as git CLI commands), so I think it warrants its own h1.